content/en/blog/2022-06-13-unikraft-releases-hyperion.md

```diff
@@ -0,0 +1,207 @@
++++
+title = "Unikraft releases v0.9.0 (Hyperion)"
+date = "2022-06-12T00:00:00+01:00"
+author = "Alexander Jung"
+tags = ["announcement"]
++++
+
+We're excited to announce Unikraft v0.9.0 (Hyperion) and to show off many of the things the community has been working on over the last two months.
+
+In this blog post, we highlight some of the new features available in Unikraft.
+A full list of features can be found [on the releases page for Hyperion](https://unikraft.org/releases/v0.9.0)
+
+### A new common SMP API
+
+_This feature was championed by [Marc Rittinghaus](https://github.com/marcrittinghaus) with extensive input from [Michalis Pappas](https://github.com/michpappas), [Răzvan Vîrtan](https://github.com/razvanvirtan) and [Simon Kuenzer](https://github.com/skuenzer)._
+
+Unikraft has introduced a new API which provides an abstraction over all the architecture-dependent hardware details.
+Already, [`arm64`](https://github.com/unikraft/unikraft/pull/373) has implemented this API with on-going work for introducing [`x86_64`](https://github.com/unikraft/unikraft/pull/244).
+This new API makes it easy to add SMP support for future architectures and standardizes the use of SMP within Unikraft for true multi-core support.
+
+Multi-core support is a heavily requested feature for Unikraft and this new API takes strides in making this a reality.
+
+#### Implementing the API for a new architecture
+
+Inside the Unikraft SMP implementation, the hardware information that is hidden by our API needs to be saved and accessed.
+This is done using an array of CPU architecture-specific data structures.
+On ARM, CPU structures are initialized during the platform setup with  information read from the device tree.
+The caller will always refer to the CPUs using logical IDs.
+These will be integers between `0` and the maximum number of cores in the system.
+The logical ids are just indexes from the SMP internal array just described.
+
+Architecture implementations must provide the following functions as described in [`/plat/common/include/uk/plat/common/lcpu.h`](#https://github.com/unikraft/unikraft/blob/RELEASE-0.9.0/plat/common/include/uk/plat/common/lcpu.) which then hook into:
```

Comment:
I think an `h` is missing at the end of the URL, like:
`#https://github.com/unikraft/unikraft/blob/RELEASE-0.9.0/plat/common/include/uk/plat/common/lcpu.h`